app/compile/template/index.js

```diff
@@ -118,7 +135,19 @@ export default async function template(files, options = {}) { // eslint-disable-
 
     let { locales: locales_to_build } = locals
     delete locals.locales
-    locales_to_build = to.array(locales_to_build, /[\s,]+/)
+
+    if (locales_to_build.includes('all')) {
+      locales_to_build = all_locales
+    }
+
+    if (locales_from_folders) {
```

Comment:
This is a more efficient way of handling this than what I did because you're only dealing with 1 locale vs potentially looping over several. 👍